cypress/e2e/shared/fluxQueryBuilder.test.ts

```diff
@@ -11,10 +11,17 @@ describe('FluxQueryBuilder', () => {
             newDataExplorer: true,
           }).then(() => {
             cy.wait(100)
-            // Switch to Flux Query Builder
-            cy.getByTestID('flux-query-builder-toggle')
-              .should('be.visible')
-              .click()
+            cy.getByTestID('flux-query-builder-toggle').then($toggle => {
+              cy.wrap($toggle).should('be.visible')
+              if ($toggle.hasClass('active')) {
+                // hasClass is a jQuery function
+                // Already switched to Flux Query Builder
+                // do nothing
+              } else {
+                // Switch to Flux Query Builder
+                $toggle.click()
```

Comment:
This is a strange way to write an `if` statement. You can keep the comments and write a different `if` statement.